src/redux/gas.ts

```diff
@@ -498,16 +558,16 @@ export const gasPricesStartPolling = (
                   type: GAS_FEES_SUCCESS,
                 });
               } catch (e) {
-                captureException(new Error('Etherscan gas estimates failed'));
-                logger.sentry('Etherscan gas estimates error:', e);
-                logger.sentry('falling back to eth gas station');
+                logger.error(
+                  new RainbowError(`Etherscan gas estimates error: ${e}`)
+                );
+                logger.debug('falling back to eth gas station');
               }
             }
             fetchResolve(true);
-          } catch (error) {
-            captureException(new Error('all gas estimates failed'));
-            logger.sentry('gas estimates error', error);
-            fetchReject(error);
+          } catch (e) {
+            logger.error(new RainbowError(`Gas Estimates Failed: ${e}`));
```

Comment:
Nit, best practice for errors: v descriptive messages. We might have other gas fetchers in the app, would be good to know that it was specifically Arbitrum, or even `getArbitrumGasPrices`, by including that in the message. Same for all these. Not a huge deal though.